javascript-practice/src/js/constants/constant.js

```diff
@@ -0,0 +1,15 @@
+export const databaseURL =
+  'https://javascript-training-81f7a-default-rtdb.asia-southeast1.firebasedatabase.app';
+export const regex =
+  /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+export const PASSWORD_NOT_MATCH = 'Password not match! Please try again!';
+export const PASSWORD_NOT_STRONG =
+  'Password must at least one uppercase, one lowercase letter, one number and one special character!';
+export const ERROR_MESSAGE_DEFAULT = 'Something went wrong!';
+export const TIME_OUT_SEC = 3;
+export const TIME_OUT_ERROR = {
+  status: 408,
+  message: 'Connection time out! Please try again!',
+};
+export const TYPE_FORM = { success: 'success', error: 'error' };
```

Comment:
Success, but error??

Reply:
Because I have 2 types of forms: success and error forms.